--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -44,7 +44,6 @@
         deps = dependencies + select({
             ":linux_x86_64": ["@qt_linux_x86_64//:qt_%s_linux" % name],
             ":linux_aarch64": ["@qt_linux_aarch64//:qt_%s_linux" % name],
-            ":osx_x86_64": ["@qt_mac_x86_64//:qt_%s_mac" % name],
             ":osx_arm64": ["@qt_mac_aarch64//:qt_%s_mac" % name],
             "@platforms//os:windows": ["@qt_windows_x86_64//:qt_%s_windows" % name],
         }),
@@ -58,7 +57,6 @@
     deps = select({
         ":linux_x86_64": ["@qt_linux_x86_64//:qt_hdrs"],
         ":linux_aarch64": ["@qt_linux_aarch64//:qt_hdrs"],
-        ":osx_x86_64": ["@qt_mac_x86_64//:qt_hdrs"],
         ":osx_arm64": ["@qt_mac_aarch64//:qt_hdrs"],
         "@platforms//os:windows": ["@qt_windows_x86_64//:qt_hdrs"],
     }),
