diff --git a/extension/qt_modules.bzl b/extension/qt_modules.bzl
--- a/extension/qt_modules.bzl	2026-10-07 04:49:14
+++ b/extension/qt_modules.bzl	2026-10-07 04:49:14
@@ -298,135 +298,135 @@
         "6.8.3": {
             "qtbase": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.clang_64/6.8.3-0-202503201723qtbase-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "c4dfc40f1be702eaa08e08cf87a749c44c55e5eb09e46a7933f18ab4b7ca7ac7",
             },
             "qtdeclarative": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.clang_64/6.8.3-0-202503201723qtdeclarative-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "9b499244b4feafc9d5ff7cdb2ed8baf8ddbb7e71f6cd828c57707e37c8327beb",
             },
             "qtsvg": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.clang_64/6.8.3-0-202503201723qtsvg-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "14f0e28662ba3d2d6b297916746e5b45197cf5cca408de7c5249ce628aecdd02",
             },
             "qt5compat": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qt5compat.clang_64/6.8.3-0-202503201723qt5compat-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "019ed04dc48c3cd92f5e37e4ce575c9de5e2bb52bdf9083ce071549505a4b523",
             },
             "qtconnectivity": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtconnectivity.clang_64/6.8.3-0-202503201723qtconnectivity-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "a17e8f4ec859176ba164798dd81ffa2bda14bc18d352ae35faa88433a8d80d9e",
             },
             "qt3d": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qt3d.clang_64/6.8.3-0-202503201723qt3d-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "27430748d04bf7f5cefd74c2fb3e7578633b73e5a7bf09890ec53281270e2758",
             },
             "qtcharts": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtcharts.clang_64/6.8.3-0-202503201723qtcharts-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "92baad004179bf3d1e24b28015140ecdfa7daf44cf46d9595d64365dd3930b7b",
             },
             "qtdatavis3d": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtdatavis3d.clang_64/6.8.3-0-202503201723qtdatavis3d-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "7ff1aa47f26fdfb2e407fa64d48ef802e0e8f83c9a1d6e8f394a9dfe48fa9f7e",
             },
             "qtmultimedia": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtmultimedia.clang_64/6.8.3-0-202503201723qtmultimedia-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "947f0871b0b18c997b4ad4fb961f6e999ef0e252c896ee88a6f0642b6b2e2633",
             },
             "qtshadertools": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtshadertools.clang_64/6.8.3-0-202503201723qtshadertools-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "89718349bad70845f023012f426e984f08d2e67c7721d82545b120a93eda63c7",
             },
             "qtquick3d": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtquick3d.clang_64/6.8.3-0-202503201723qtquick3d-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "5173ea13349b648944c2a26fe56f68a0ab5b8ef297ae44dc2d38d63bc7f59227",
             },
             "qtquicktimeline": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtquicktimeline.clang_64/6.8.3-0-202503201723qtquicktimeline-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "b247df9babd7eee4fbf84ceaec4a298c2cc44c5118c741942e3a6c69ea050376",
             },
             "qtlottie": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtlottie.clang_64/6.8.3-0-202503201723qtlottie-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "5a920584c7687f328210eac4a01c66cc6919eea6422671b9d62665d67b86e494",
             },
             "qtscxml": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtscxml.clang_64/6.8.3-0-202503201723qtscxml-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "8790ca2392adfc50e4a713079aad89d84c56f6f1935174b65c19c3b8640027c8",
             },
             "qtremoteobjects": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtremoteobjects.clang_64/6.8.3-0-202503201723qtremoteobjects-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "16f133f31dfd87293fe3eb5685eabcb750c59c2760f945543a30bcf76c63eef7",
             },
             "qtserialport": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtserialport.clang_64/6.8.3-0-202503201723qtserialport-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "9fe96afd94c87cf952bbf4bfdfb2f13fc66afef8ae70d4b9f96ad5c740715c8f",
             },
             "qtserialbus": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtserialbus.clang_64/6.8.3-0-202503201723qtserialbus-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "fd2a423974f8464f8ce55f8205217f6835577c75d77917dbe94a915f0f8f9957",
             },
             "qtsensors": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtsensors.clang_64/6.8.3-0-202503201723qtsensors-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "ee0e87ff12e1c896e47cd4a8de2c514deabb4f61bdf3cee54d292bf29dca328d",
             },
             "qtpositioning": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtpositioning.clang_64/6.8.3-0-202503201723qtpositioning-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "f3804682a1b1d5c52a8309e519a2b7ad201d0750a18c3d8cfdc3f9300ed05f33",
             },
             "qtnetworkauth": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtnetworkauth.clang_64/6.8.3-0-202503201723qtnetworkauth-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "64a5c2a8016476d2c224a7158d12a63878881d8524a2f87f45a28408ad223dda",
             },
             "qtimageformats": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtimageformats.clang_64/6.8.3-0-202503201723qtimageformats-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "aa22d0b0bb861f4ad54c7ee59e5f7728567c2a845b4d53b2a1d6550a6047c7a0",
             },
             "qthttpserver": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qthttpserver.clang_64/6.8.3-0-202503201723qthttpserver-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "855995ad9756e4cdd42a0955ad5180ed1ec45f0eb2e11abe375e076d9c290aeb",
             },
             "qtlanguageserver": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtlanguageserver.clang_64/6.8.3-0-202503201723qtlanguageserver-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "7bdb7c4810972a3c3d4312663c18a0aeb3f0f27a0d14291cb09bd95d37b2e704",
             },
             "qtgrpc": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtgrpc.clang_64/6.8.3-0-202503201723qtgrpc-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "eccfbd62eff19fd60e004ec0180ad21bd3a1d21abd3a7d54a31b5f6ec5028e35",
             },
             "qtquick3dphysics": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtquick3dphysics.clang_64/6.8.3-0-202503201723qtquick3dphysics-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "5d10965cf3b0be489f0e1d53a4f89a839656470218c4dd195437fffa7a37ac6c",
             },
             "qtspeech": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtspeech.clang_64/6.8.3-0-202503201723qtspeech-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "ad9167f78d7a000c6f2b11b7a7936bd9d02397d7b7ecd917b7c55586cbb7e0a5",
             },
             "qtlocation": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtlocation.clang_64/6.8.3-0-202503201723qtlocation-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "3b26ce0bdb3f131f4eda28ca986d949fdf7154209de33e459ed461ae79191acc",
             },
             "qtquickeffectmaker": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtquickeffectmaker.clang_64/6.8.3-0-202503201723qtquickeffectmaker-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "211fc3c9c66c6306f51d7d61fe419ba7052c4bf2b106006f05ba505d6896938b",
             },
             "qtgraphs": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtgraphs.clang_64/6.8.3-0-202503201723qtgraphs-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "411dfb884581394e727bfe8bf344576c7525540b6e14f40b33a3860f3e08a4af",
             },
             "qtvirtualkeyboard": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtvirtualkeyboard.clang_64/6.8.3-0-202503201723qtvirtualkeyboard-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "a9de0c6ed5a628cfe2ba88c2ec82e0a5374aa12c3c5a0cb70ade6f8fcf288316",
             },
             "qtwebchannel": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtwebchannel.clang_64/6.8.3-0-202503201723qtwebchannel-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "71b339c6668ff69ed771f62ac669561d3cf00704bb5ab78b2e82ba034c013b67",
             },
             "qtwebsockets": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtwebsockets.clang_64/6.8.3-0-202503201723qtwebsockets-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "d4d903cd291f3ac2c2c4bb0a515900c0567e8e6e9dffbe24a188518cd294e2d7",
             },
             "qtwebview": {
                 "url": QT_BASE_DOWNLOAD_URL + "mac_x64/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtwebview.clang_64/6.8.3-0-202503201723qtwebview-MacOS-MacOS_14-Clang-MacOS-MacOS_14-X86_64-ARM64.7z",
-                "sha256": "",
+                "sha256": "ffddb95ca4732bc7d91026d7a23f19f489aaf6f5d029b7a2e5ec00d4fcc1da95",
             },
         },
     },
@@ -448,35 +448,35 @@
         "6.8.3": {
             "qtbase": {
                 "url": QT_BASE_DOWNLOAD_URL + "windows_x86/desktop/qt6_683/qt6_683/qt.qt6.683.win64_msvc2022_64/6.8.3-0-202503201308qtbase-Windows-Windows_11_23H2-MSVC2022-Windows-Windows_11_23H2-X86_64.7z",
-                "sha256": "",
+                "sha256": "41688269fac0565db956c66d9eecae777d16197e0c02cd81b88640c1f5d73d3f",
             },
             "qtdeclarative": {
                 "url": QT_BASE_DOWNLOAD_URL + "windows_x86/desktop/qt6_683/qt6_683/qt.qt6.683.win64_msvc2022_64/6.8.3-0-202503201308qtdeclarative-Windows-Windows_11_23H2-MSVC2022-Windows-Windows_11_23H2-X86_64.7z",
-                "sha256": "",
+                "sha256": "c2f9f481bf6e8fb004530f20b820fbc77d4ee6dcca24d1ea1d1863d99c5dabf1",
             },
             "qtcharts": {
                 "url": QT_BASE_DOWNLOAD_URL + "windows_x86/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtcharts.win64_msvc2022_64/6.8.3-0-202503201308qtcharts-Windows-Windows_11_23H2-MSVC2022-Windows-Windows_11_23H2-X86_64.7z",
-                "sha256": "",
+                "sha256": "ed75d36682042c2e4a97576b0b8f92fdb8ef43e40489566e4e44fc469ee7903c",
             },
             "qtremoteobjects": {
                 "url": QT_BASE_DOWNLOAD_URL + "windows_x86/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtremoteobjects.win64_msvc2022_64/6.8.3-0-202503201308qtremoteobjects-Windows-Windows_11_23H2-MSVC2022-Windows-Windows_11_23H2-X86_64.7z",
-                "sha256": "",
+                "sha256": "86c39fa3495a2057c9c748209e22bc76fa3af535a3cc6d2564cee8995a683b2d",
             },
             "qtserialport": {
                 "url": QT_BASE_DOWNLOAD_URL + "windows_x86/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtserialport.win64_msvc2022_64/6.8.3-0-202503201308qtserialport-Windows-Windows_11_23H2-MSVC2022-Windows-Windows_11_23H2-X86_64.7z",
-                "sha256": "",
+                "sha256": "7a11b8b7d38edc026959a4b115ecec1ffe750a7059c03b40e9f7c99ae95e7b7e",
             },
             "qtwebsockets": {
                 "url": QT_BASE_DOWNLOAD_URL + "windows_x86/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qtwebsockets.win64_msvc2022_64/6.8.3-0-202503201308qtwebsockets-Windows-Windows_11_23H2-MSVC2022-Windows-Windows_11_23H2-X86_64.7z",
-                "sha256": "",
+                "sha256": "b38bd17e47554c17bf5e85fb087e54c394128e70769f206381247097894ebdbf",
             },
             "qtsvg": {
                 "url": QT_BASE_DOWNLOAD_URL + "windows_x86/desktop/qt6_683/qt6_683/qt.qt6.683.win64_msvc2022_64/6.8.3-0-202503201308qtsvg-Windows-Windows_11_23H2-MSVC2022-Windows-Windows_11_23H2-X86_64.7z",
-                "sha256": "",
+                "sha256": "48d1d798894cbf8696a0010fcfdeafea652b35159c56ea2060ee85ce3d0048f7",
             },
             "qt5compat": {
                 "url": QT_BASE_DOWNLOAD_URL + "windows_x86/desktop/qt6_683/qt6_683/qt.qt6.683.addons.qt5compat.win64_msvc2022_64/6.8.3-0-202503201308qt5compat-Windows-Windows_11_23H2-MSVC2022-Windows-Windows_11_23H2-X86_64.7z",
-                "sha256": "",
+                "sha256": "415103bfb3d23e057f4c99062a7985ccedf76d23705d6c580ffd76e363d24439",
             },
         },
     },
